src/Runtime/ExecutionSession.cpp

```diff
@@ -41,6 +41,25 @@ ExecutionSession::ExecutionSession(
     errStr << "Cannot load symbol: '" << entryPointName << "'" << std::endl;
     throw std::runtime_error(errStr.str());
   }
+
+  std::string inputSignatureName("omInputSignature");
```

Comment:
Also, I think it would be better to have their names as const members of the class, for example, rather than hardcoded here.

Reply:
done